controllers/discordactions.js

```diff
@@ -409,15 +409,17 @@ const setRoleToUsersWith31DaysPlusOnboarding = async (req, res) => {
 
 const generateInviteForUser = async (req, res) => {
   try {
-    const { userId } = req.query;
+    const { userId, dev } = req.query;
     const userIdForInvite = userId || req.userData.id;
+    let inviteLink = "";
 
-    const modelResponse = await discordRolesModel.getUserDiscordInvite(userIdForInvite);
-
-    if (!modelResponse.notFound) {
-      return res.status(409).json({
-        message: "User invite is already present!",
-      });
+    if (!dev) {
```

Comment:
will this execute?

Reply:
How?

Reply:
What do you mean by execute? Do you mean can we run this API to check the response ?

Reply:
What will be the value of dev?

Reply:
Will it execute for dev='false'?